docs/src/scss/_typography.scss

```diff
@@ -34,6 +34,10 @@ li {
   color: $paragraph-color;
 }
 
+.prism-code * {
+  color: inherit;
```

Comment:
Is this inherit property needed here ?

Reply:
yes, since this property `https://github.com/openebs/website/blob/revamp-3.0/docs/src/scss/_typography.scss#L31`  is making color change everywhere including `prism-code class`.